turbo.json

```diff
@@ -6,6 +6,10 @@
       "dependsOn": ["^build"],
```

Comment:
I think we need to add
    "@cloudoperators/juno-communicator": "*",
    "@cloudoperators/juno-messages-provider": "*",
    "@cloudoperators/juno-ui-components": "*",
    "@cloudoperators/juno-url-state-provider-v1": "^1.3.2",
    "@cloudoperators/juno-utils": "*",
    all of this packages on to dependencies in turbo to avoid possible race codnitions 
    @andypf  corercts me if I am wrong?

Reply:
you are right

Reply:
This is the case @andypf Please check the file:
https://github.com/cloudoperators/juno/blob/artie-doop-vite/apps/doop/turbo.json

Reply:
Yea cool, but we need another section for `build:static`. You can just copy it.

Reply:
right!! let me then fix it.

Reply:
right!! let me then fix it.